datalad/support/sshconnector.py

```diff
@@ -207,6 +215,12 @@ def runner(self):
             self._runner = WitlessRunner()
         return self._runner
 
+    @property
+    def ssh_version(self):
+        if self._ssh_version is None:
+            self._ssh_version = external_versions["cmd:ssh"].version
```

Comment:
what happens if `ssh` is not found -- would `.version` become `None` and thus making us to reask it every time?